rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -439,6 +439,28 @@
             (BridgeMethodExecutorTyped) Bridge::getStateForDebugging,
             true
     ),
+    GET_LOCKING_CAP(
+            CallTransaction.Function.fromSignature(
+                    "getLockingCap",
+                    new String[]{},
+                    new String[]{"int256"}
```

Comment:
Why is the argument to `INCREASE_LOCKING_CAP` an `int256` and not an `uint256`? it makes more sense to me :thinking:

Reply:
There's clearly an old bug in ethereumj which answers my question: https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/org/ethereum/solidity/SolidityType.java#L67